src/pt/portugamangas/src/eu/kanade/tachiyomi/extension/pt/portugamangas/PortugaMangas.kt

```diff
@@ -0,0 +1,182 @@
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = "$baseUrl".toHttpUrl().newBuilder()
+            .addQueryParameter("pagina", "$page")
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun latestUpdatesSelector() = "div.atualizacao"
+
+    override fun latestUpdatesFromElement(element: Element): SManga = SManga.create().apply {
+        title = element.selectFirst("a > h3")?.text() ?: "Title not found"
+        thumbnail_url = element.selectFirst("div > img")!!.absUrl("src")
+        setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
+    }
+
+    override fun latestUpdatesNextPageSelector(): String? = "ul.pagination [aria-label=Next]"
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$baseUrl/mangas".toHttpUrl().newBuilder()
+            .addQueryParameter("busca", query)
+            .addQueryParameter("pagina", "$page")
+            .build()
+        return GET(url, headers)
+    }
+
+    private fun hasNextPage(currentPage: Int, document: Document): Boolean {
+        val pageAmount = document.select("ul.pagination li:not([aria-hidden])").size
+        return pageAmount != currentPage && currentPage != -1
+    }
+
+    private fun searchMangaParse(document: Document): List<SManga> =
+        document.select(searchMangaSelector()).map { this.searchMangaFromElement(it) }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val currentPage = response.request.url.queryParameter("pagina")?.toInt() ?: -1
+        val document = response.parseAsDocument()
+        return MangasPage(searchMangaParse(document), hasNextPage(currentPage, document))
+    }
+
+    override fun searchMangaSelector(): String = "div.mangas"
+
+    override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
+        title = element.selectFirst("h3")?.ownText() ?: "Untitled"
+        thumbnail_url = element.selectFirst(searchMangaThumbnailSelector())?.attr("src")?.toAbsURL()
+        setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
+    }
+
+    private fun searchMangaThumbnailSelector(): String = "img"
+
+    override fun searchMangaNextPageSelector(): String? = "ul.pagination li:last-child"
+
+    override fun getMangaUrl(manga: SManga): String = baseUrl + manga.url
+
+    private fun getMangaStatus(document: Document): String =
+        document.selectFirst("h5.cg_color > a.label.label-success")?.text() ?: ""
+
+    override fun mangaDetailsParse(document: Document): SManga {
+        return SManga.create().apply {
+            title = document.selectFirst("h1")?.text() ?: "Untitled"
+            description = document.selectFirst("#manga_capitulo_descricao")?.text()
+            thumbnail_url = document.selectFirst("div.manga .row .row div.text-right img")!!.absUrl("src")
+            update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
```

Comment:
are you sure you need this? This will cause app to not include the manga in global update as it is intended for extensions which only provide a single chapter or "gallery"